### ISSUE TITLE
command "rails generate scaffold post title:string body:text" is stuck

### ISSUE THREAD

**Issue by zendevil** [rails/rails#36515]
In the root directory of my project, I'm running the following command:

```
rails generate scaffold post title:string body:text
```

But it is just stuck there. This is Rails 6.0.0.rc1

**Comment by zendevil** [comment]
Yeah, 6.0.0.rc1 is not quite there yet. The command runs fine on 5.2.3

**Comment by rafaelfranca** [comment]
Can you provide an example application that reproduce the issue?

**Comment by pbzona** [comment]
I ran into a similar issue with 6.0.0.rc1 after I manually deleted some generated files (rather than destroying the scaffold via CLI) and tried to re-generate them. 

What fixed the issue for me was deleting the `bin/` directory and resetting the binstubs via `rake app:update:bin`. Note that you use rake here since the project's `rails` is temporarily gone.

Not sure what the root cause might be, but hopefully this helps.
